bin/counterfact.js

```diff
@@ -97,17 +178,19 @@ async function main(source, destination) {
     "🎉 VERSION 1.0 IS COMING! LEARN MORE:",
     "   https://github.com/pmcelhaney/counterfact/issues/823",
     "",
-    "Starting REPL, type .help for more info",
+    watchMessage,
+    config.startServer ? "Starting server" : undefined,
```

Comment:
Do you think we need these messages? They've been there forever, since before adding the big banner, and I wonder if they're still necessary. 

I'll leave it up to you whether to move the construction of `watchMessage` to a function, which will lower the cognitive complexity, or delete it altogether.

Reply:
I refactored to a function and left the messages for now. It seemed helpful for the user to know that the options they chose are executing, but that said, if we just want to add `debug('generating types')` lines instead of adding to the welcome banner I'm fine with that.